Sources/Subscription/AccountManager.swift

```diff
@@ -182,6 +207,10 @@ public class AccountManager: AccountManaging {
         NotificationCenter.default.post(name: .accountDidSignIn, object: self, userInfo: nil)
     }
 
+    public func signOut() {
+        signOut(skipNotification: false)
```

Comment:
defaulting skipNotification to false on this method will make these calls cleaner.